src/components/PricingCard/PricingCard.tsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import styled from 'styled-components';
+import CheckmarkIcon from '../../images/svg/checkmark.svg';
+import TalkIcon from '../../images/svg/talk.svg';
+import { Link } from '@material-ui/core';
+
+type PricingCardProps = {
+  color: string;
+  buttonText: string;
+  buttonLink: string;
+  title: string;
+  price?: string;
+  denominator?: string;
+  isMostPopular?: boolean;
+  benefits: string[] | string;
+};
+
+const TitleComponent = styled.div`
+  font-weight: 900;
+  font-size: 28px;
+  margin: 10px 0px;
+`;
+
+const PriceComponent = styled.div`
+  font-size: 28px;
+  margin: 10px 0px;
+`;
+
+const TalkIconComponent = styled.div`
+  width: 45px;
+  height: 45px;
+  & svg {
+    width: 100%;
+    height: 100%;
+  }
+`;
+
+const DenominatorComponent = styled.span`
+  font-size: 14px;
+  font-weight: 100;
+  vertical-align: super;
+  letter-spacing: 0.5px;
+`;
+
+const BulletpointWrapComponent = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 10px;
+`;
+
+const LinkComponent = styled(Link)`
+  &:hover {
+    text-decoration: none !important;
+  }
+`;
+
+export const PricingCard = ({
+  color,
+  title,
+  price,
+  buttonText,
+  benefits,
+  denominator,
+  isMostPopular,
+  buttonLink,
+}: PricingCardProps) => {
+  const isBenefitsList = typeof benefits === 'string';
+
+  const CardComponent = styled.div`
+    position: relative;
+    display: flex;
+    flex-direction: column;
+    border: 2px solid ${color};
+    border-radius: 36px;
+    width: 100%;
+    padding: 24px;
+  `;
+
+  const BadgeComponent = styled.div`
+    position: absolute;
+    left: 50%;
+    top: 0px;
+    transform: translate(-50%, -50%);
+    font-size: 14px;
+    padding: 8px 32px;
+    border-radius: 32px;
+    font-weight: 500;
+    background-color: ${color};
+    white-space: nowrap;
+    color: white;
+  `;
+
+  const BenefitsComponent = styled.div`
+    text-align: ${isBenefitsList ? 'center' : 'lefft'};
+    display: flex;
+    flex-flow: column;
+    flex-grow: 1;
+    gap: 15px;
+    justify-content: center;
+    padding: 15px 0px;
+    margin-bottom: 15px;
+  `;
+
+  const BulletPointComponent = styled.div`
+    color: ${color};
+    width: 20px;
+    height: 20px;
+    flex-shrink: 0;
+    & svg {
+      width: 100%;
+      height: 100%;
+    }
+  `;
+
+  const ButtonComponent = styled.button`
+    background-color: ${color};
+    display: block;
+    margin: auto;
+    color: white;
+    border-radius: 30px;
+    border: 0px;
+    padding: 15px 30px;
+    font-weight: 700;
+    font-size: 18px;
+    cursor: pointer;
+  `;
```

Comment:
i'd rather move this definitions out of component body, [passing props to styled components](https://styled-components.com/docs/basics#adapting-based-on-props)